__tests__/decorators.test.ts

```diff
@@ -219,7 +218,9 @@ describe('decorators', () => {
       }
     }
 
-    routes = _.keyBy(parseRoutes(getMetadataArgsStorage()), 'action.method')
+    parseRoutes(getMetadataArgsStorage()).forEach(
+      (route) => (routes[route.action.method] = route)
+    )
```

Comment:
nit: could use `.reduce` here to match previous behaviour

Reply:
`reduce` instead of `forEach` 75495ff